annotation_tool_example.ipynb

```diff
@@ -0,0 +1,71 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "2fce4540-ec7d-44ec-a65d-2be55e9ce758",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from annotation_tool import Page"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0932af45-196b-46f8-af56-7821df40b048",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "929792bf0e4743e0ab5e29b58c70fa4c",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/html": [
+       "Cannot show widget. You probably want to rerun the code cell above (<i>Click in the code cell, and press Shift+Enter <kbd>⇧</kbd>+<kbd>↩</kbd></i>)."
+      ],
+      "text/plain": [
+       "Cannot show ipywidgets in text"
+      ]
```

Comment:
Not sure if output not saved/updated or if it simply cannot be rendered offline?